runtime/compiler/optimizer/MethodHandleTransformer.cpp

```diff
@@ -350,9 +350,15 @@ TR_MethodHandleTransformer::getObjectInfoOfNode(TR::Node* node)
    if (node->getOpCode().isLoadDirect() &&
        symbol->isAutoOrParm())
       {
+      TR::KnownObjectTable::Index koi = (*_currentObjectInfo)[symbol->getLocalIndex()];
+      node->setKnownObjectIndex(koi);
```

Comment:
@gita-omr fyi since you had wanted to see known object index on node (or at least somewhere more expansive than on the sym ref) for some vector work as well.